docs/components/Datepicker/Web.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { DatePicker } from "@jobber/components/DatePicker";
+import { Content } from "@jobber/components/Content";
+import { DataDump } from "@jobber/components/DataDump";
+import { Button } from "@jobber/components/Button";
+
+export default {
+  title: "Components/Selections/DatePicker/Web",
+  component: DatePicker,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof DatePicker>;
+
+const BasicTemplate: ComponentStory<typeof DatePicker> = args => {
+  const [date, setDate] = useState(new Date("01/01/2021"));
+  return (
+    <Content>
+      <DatePicker {...args} selected={date} onChange={setDate} />
+      {date && <DataDump defaultOpen data={date} />}
+    </Content>
+  );
+};
+
+export const Basic = BasicTemplate.bind({});
+Basic.args = {};
+
+export const Inline = BasicTemplate.bind({});
+Inline.args = {
+  inline: true,
+};
+
+const MinMaxTemplate: ComponentStory<typeof DatePicker> = args => {
+  const [date, setDate] = useState(new Date("01/01/2021"));
+
+  return (
+    <Content>
+      <DatePicker
+        {...args}
+        minDate={new Date("02/07/2022")}
```

Comment:
Since it is the purpose of the example, consider making these part of the `args` so the user can manipulate them in the Canvas section...

Reply:
The issue is that args doesn't work with `Date` objects. Other examples do the same thing because of this

Reply:
Doh, thought you might say that. Ok. :/